src/model/BoilerHotWater.cpp

```diff
@@ -478,7 +475,7 @@ namespace model {
 
     setBoilerFlowMode("ConstantFlow");
 
-    setParasiticElectricLoad(0.0);
+    setOnCycleParasiticElectricLoad(0.0);
```

Comment:
I suggest changing the IDD and making it required-field instead of default 0.0, while we're at it